index.html

```diff
@@ -8,13 +8,24 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
+      <header>
+          <nav class="nav-head">
+              <h2>Awesome Books</h2>
+              <ul class="nav-item">
+                  <li class="nav-link"><a href="#booklist">List</a></li>
+                  <li class="nav-link"><a href="#addBook">Add new</a></li>
+                  <li class="nav-link"><a href="#contact">Contact</a></li>
+              </ul>
+          </nav>
+      </header>
+      <div id="current-date"></div>
```

Comment:
- [ ] Kindly make sure that when any of the navigation links are clicked, the main area changes to display only one of the three content sections. 
- That means that the main section shouldn't contain all sections to be apparent at any time. The main section should only display one of the three sections according to which navigation link was clicked.
- In other words, you're trying to simulate a single-page application where content is displayed dynamically according to user interaction.
- [HINT] You can do so by using the `display` property to control which content is hidden and which content is displayed.

![image](https://user-images.githubusercontent.com/62290545/171700415-6a01e2af-77e6-4bfe-bc34-735fa90b1f66.png)